app/src/main/java/org/jellyfin/androidtv/ui/livetv/TvManager.java

```diff
@@ -143,59 +131,22 @@ public void onResponse(final ChannelInfoDtoResult response) {
                 }
 
-                outerResponse.onResponse(sortChannels());
+                outerResponse.onResponse(fillChannelIds());
             }
         });
-
     }
 
     private static int fillChannelIds() {
-        int ndx = 0;
-        channelIds = new String[allChannels.size()];
-        String last = getLastLiveTvChannel();
-        int i = 0;
-        for (ChannelInfoDto channel : allChannels) {
-            channelIds[i++] = channel.getId();
-            if (channel.getId().equals(last)) ndx = i;
-            //TvApp.getApplication().getLogger().Debug("Last played for "+channel.getName()+ " is "+channel.getUserData().getLastPlayedDate());
-        }
-
-        return ndx;
-    }
-
-
-    public static int sortChannels() {
-        LiveTvPreferences liveTvPreferences = get(LiveTvPreferences.class);
-
         int ndx = 0;
         if (allChannels != null) {
-            //Sort by last played - if selected
-            if (ItemSortBy.DatePlayed.equals(liveTvPreferences.get(LiveTvPreferences.Companion.getChannelOrder()))) {
-                Collections.sort(allChannels, Collections.reverseOrder(new Comparator<ChannelInfoDto>() {
-                    @Override
-                    public int compare(ChannelInfoDto lhs, ChannelInfoDto rhs) {
-                        long left = lhs.getUserData() == null || lhs.getUserData().getLastPlayedDate() == null ? 0 : lhs.getUserData().getLastPlayedDate().getTime();
-                        long right = rhs.getUserData() == null || rhs.getUserData().getLastPlayedDate() == null ? 0 : rhs.getUserData().getLastPlayedDate().getTime();
-
-                        long result = left - right;
-                        return result == 0 ? 0 : result > 0 ? 1 : -1;
-                    }
-                }));
-            }
-
-            if (liveTvPreferences.get(LiveTvPreferences.Companion.getFavsAtTop())) {
-                Collections.sort(allChannels, new Comparator<ChannelInfoDto>() {
-                    @Override
-                    public int compare(ChannelInfoDto channelOne, ChannelInfoDto channelTwo) {
-                        boolean channelOneFav = channelOne.getUserData() != null && channelOne.getUserData().getIsFavorite();
-                        boolean channelTwoFav = channelTwo.getUserData() != null && channelTwo.getUserData().getIsFavorite();
-                        return -Boolean.compare(channelOneFav, channelTwoFav);
-                    }
-                });
+            channelIds = new String[allChannels.size()];
+            String last = getLastLiveTvChannel();
+            int i = 0;
+            for (ChannelInfoDto channel : allChannels) {
+                channelIds[i++] = channel.getId();
+                if (channel.getId().equals(last)) ndx = i;
+                //TvApp.getApplication().getLogger().Debug("Last played for "+channel.getName()+ " is "+channel.getUserData().getLastPlayedDate());
```

Comment:
This comment can be removed (or changed to a Timber.d call)

Reply:
:+1: All commented logging lines have been removed from this file now.